File: Programming Basics/Programming Basics - C#/Exercises/04. Complex Conditionals/04. Complex Conditionals/Cinema/Cinema.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Cinema
{
    class Program
    {
        static void Main(string[] args)
        {
            string types = Console.ReadLine().ToLower();
            int rows = int.Parse(Console.ReadLine());
            int columns = int.Parse(Console.ReadLine());

            int full = rows * columns;
            double income = -1;

            switch (types)
            {
                case "premiere":
                    income = full * 12.00;
                    break;
                case "normal":
                    income = full * 7.50;
                    break;
                case "discount":
                    income = full * 5.00;
                    break;
                default:
                    break;
            }

            Console.WriteLine(Math.Round(income, 2) + " leva");
        }
    }
}

File: Programming Basics/Programming Basics - JavaScript/Exercises/04. Complex Conditions/11. Cinema/Cinema.js
function calculateCinemaIncome([arg1, arg2, arg3]) {
    let type = arg1;
    let rows = Number(arg2);
    let cols = Number(arg3);

    let income;
    switch (type) {
        case "Premiere": {
            income = 12 * rows * cols;
            break;
        }
        case "Normal": {
            income = 7.50 * rows * cols;
            break;
        }
        case "Discount": {
            income = 5 * rows * cols;
        }
    }
    console.log(`${income.toFixed(2)} leva`);
}

/* Uncomment to test:
calculateCinemaIncome(["Normal", "21", "13"]);
    */